pipeline/README.md

```diff
@@ -0,0 +1,47 @@
+# OpenShift/Jenkins Pipeline
+
+Collection of groovy scripts to help run jobs in a Jenkins pipeline.
+
+This folder contains following:
+- gradle : Directory containing gradle dependencies and runtime wrapper jar
+- src/groovy : Groovy files containing the runtime code for different build steps in the pipeline
+- build.gradle : Build configuration for the gradle runner
+- config.groovy : Configuration file containing information regarding different deployment environments in the pipeline
+- gradlew(.bat) : Gradle runner executable
+
+Do not put any confidential/secret key into the config.groovy file as it gets checked
+into the git repository.
+
+## Library Dependency
+The scripts all inherit base classes from the [BCDevops/ocp-cd-pipeline](https://github.com/BCDevOps/ocp-cd-pipeline)
+
+
+## Github Flow
```

Comment:
The merge model refers to GitHub Flow, but links to Git Flow, which is different.  Likely L19 is a typo.